README.md

```diff
@@ -185,6 +185,11 @@ Commonly enabled services that you may wish to disable:
  * pflogd
  * sndiod
 
+Note that Krun is unable to check whether turbo mode is enabled on OpenBSD or
+not, and is also unable to use APERF/MPERF ratios to indirectly check whether
```

Comment:
Kill "or not"?

Reply:
I don't think the sentence parses without "or not" ("whether" requires two clauses).

Reply:
Both are fine for me, so as you wish.

Reply:
Let's keep it as-is. I think we can then merge?